database/events_schema.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const eventsSchema = mongoose.Schema({
+  eventTitle: {type: String, required: true},
+  eventDate: {type: Date, min: Date.now(), required},
```

Comment:
minimum date they can type in, it should be minimum the date of the current day they add the event in